src/Core/Components/Base/IFluentRadioValueProvider.cs

```diff
@@ -0,0 +1,10 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+internal interface IFluentRadioValueProvider
```

Comment:
This interface is only used with the FluentRadio elements. Probably better to move this file in the `Radio` folder